### PR TITLE
Test iPad core experience for fronts

### PR DESCRIPTION
**Pull request by JustinPinner**
We want to test if serving the core version of fronts to half of our iPad 3 traffic helps prevent crashing on those devices, and we'll monitor the effects using cloudwatch beacons.

Based on recent traffic figures, we expect the test to occur on just under 1.4M requests for fronts over a two week period.

Because it wasn't possible to set up a true A/B test for this, users are equally likely to see the core or  premium mode fronts at each request.

Finally, the test applies to all fronts, but we're currently only sending beacons on each region's main front (uk, us and au) for consistency with previous monitoring.

@michaelwmcnamara and I have run this through on a few real devices and we're happy that it's doing what it's supposed to.

@sndrs, this look ok to you too?

### CODE REVIEW

**Comment by michaelwmcnamara** [comment]
QA +1 - tested the beaconing on this for all both the core experiment and for the standard beacons; tested the code opting users into the test - all looks good.

File: common/app/templates/headerInlineJS/cloudwatchBeacons.scala.js [hunk 1]
```diff
@@ -53,21 +88,23 @@
...
+        //        logDevice('nexus5', 'android');
+        //    }
+        //}
+        //
+        //// Send AWS beacons for Windows7 Chrome & Opera users (user agent is almost identical). This is so we have a indicator of loss on something which we are confident is stable
+        //if (navigator.platform === 'Win32') {
+        //    if ((/Windows NT 6.1; WOW64/.test(navigator.userAgent)) && (/Chrome/.test(navigator.userAgent))) {
+        //        logDevice('chrome', 'windows7');
+        //    }
+        //}
```
Comment (gklopper) [inline common/app/templates/headerInlineJS/cloudwatchBeacons.scala.js]: Maybe use the "Twirl" comments...

    @*
        This is a comment
    *@

That way it gets compiled out of the template.

File: common/app/views/fragments/head.scala.html [hunk 2]
```diff
@@ -14,6 +14,22 @@
     <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
 <![endif]-->
 
+@* provide some ipad users with the core view of fronts *@
+<script>
```
Comment (gklopper) [inline common/app/views/fragments/head.scala.html]: Does this "random" choice persist? Will people get the same thing every time they show up?

Comment (JustinPinner) [inline common/app/views/fragments/head.scala.html]: It's not persisted. They may see core or full versions on each view.

Comment (gklopper) [inline common/app/views/fragments/head.scala.html]: That would not be a good test then, one major question to ask is "Do people who see the core version come back more often?"

Comment (gklopper) [inline common/app/views/fragments/head.scala.html]: Drop a cookie...

Comment (sndrs) [inline common/app/views/fragments/head.scala.html]: :+1: @gklopper

Comment (sndrs) [inline common/app/views/fragments/head.scala.html]: or ideally localstorage, since iPads will have it

**Comment by sndrs** [comment]
:+1: apart from persisting the selection.

would it be worth holding off merging till I've got the core fonts stuff in? that way it should be much less of difference. of course it wouldn't be testing how much of a difference the lack of fonts makes to core...

**Comment by JustinPinner** [comment]
Thanks @sndrs I'll investigate persisting the participation. It could probably wait for your fonts change, but that could go in afterwards anyway I think?

**Comment by sndrs** [comment]
> It could probably wait for your fonts change, but that could go in afterwards anyway I think?

yeah no technical reason to delay it, just brings the two versions into visual parity (i.e. all you really lose are features)

**Comment by sndrs** [comment]
fyi, fonts for core are now in PR #10549

**Comment by michaelwmcnamara** [comment]
QA+1 for the second round

**Pull request merged**